src/main/models/roles/GuildRole.java

```diff
@@ -9,4 +18,83 @@ public class GuildRole {
 	// public static GuildRole getByAccountId(Connection con, String account_id) {
 	//
 	// }
+
+	public GuildRole(String id, String guild_id, String name) {
+		this.id = id;
+		this.guild_id = guild_id;
+		this.name = name;
+	}
+
+	public String getId() {return id;}
+
+	public String getGuild_id() {return guild_id;}
+
+	public String getName() {return name;}
+
+	public int getAccountId(Connection con, GuildData data) throws SQLException, NotFoundException {
+		String query = "SELECT * FROM user_account WHERE username = ?";
+
+		PreparedStatement stmt = con.prepareStatement(query);
+
+		stmt.setString(1, data.getUsername());
+		ResultSet rs = stmt.executeQuery();
+		if(!rs.next())
+			throw new NotFoundException("Account ID doesn't exist!");
+		return rs.getInt("id");
+	}
+
+	public int getGuildId(Connection con, GuildData guildData) throws SQLException, NotFoundException{
+		String query = "SELECT * FROM guild WHERE name = ?";
```

Comment:
- Why the name function is getGuildId but you select * ?
- What about if I use many continuous method ? (.getGuildId().insert()...) We need to put args again, right ?